src/components/data-management/ProjectDetails.jsx

```diff
@@ -611,6 +614,7 @@ const ProjectDetails = ({ width, height }) => {
           pipelineStatus.ABORTED,
           pipelineStatus.TIMED_OUT,
           pipelineStatus.FAILED,
+          pipelineStatus.SUCCEEDED,
```

Comment:
Is this something that we want to leave on while https://biomage.atlassian.net/browse/BIOMAGE-1044 is not solved?

Reply:
No, this was a shortcut so that I can trigger the pipeline repeatedly in the UI. Removed.